app/views/request/_attachments.html.erb

```diff
@@ -0,0 +1,39 @@
+<% attachments = incoming_message.get_attachments_for_display %>
+<% if not attachments.nil? and attachments.size > 0 %>
+  <div class="attachments">
+    <div class="attachments__header">
+      <h3>
+        <%= n_('{{count}} Attachment',
+               '{{count}} Attachments',
+               attachments.size,
+               :count => attachments.size) %>
+      </h3>
+    </div>
+
+    <ul class="list-of-attachments">
+      <% attachments.each do |a| %>
+        <%= tag.li class: 'attachment', id: dom_id(a) do %>
+          <%= attachment_link(incoming_message, a) %>
+
+          <p class="attachment__name">
+            <%= h a.display_filename %>
+          </p>
+
+          <p class="attachment__meta">
+            <%= a.display_size %>
+            <%= link_to "Download", attachment_path(a) %>
+            <% if a.has_body_as_html? && incoming_message.info_request.prominence(:decorate => true).is_public? %>
+              <%= link_to "View as HTML", attachment_path(a, :html => true) %>
+            <% end %>
+            <%= a.extra_note %>
+          </p>
+        <% end %>
+      <% end %>
+    </ul>
+
+    <a href="#" class="attachments__show-more" style="display: none;" data-show-all="<%= _('Show all attachments') %>" data-show-fewer="<%= _('Show fewer attachments') %>"></a>
+  </div>
+<% end %>
+
+<%= tag.div incoming_message.get_body_for_html_display(@collapse_quotes),
```

Comment:
Should this be rendered in `app/views/request/_incoming_correspondence.html.erb` below the `render partial: 'request/attachments'` call?

I guess this is technically an attachment… but conceptually… IDK?

Reply:
Well I could've left the partial named `bubble` then this wouldn't be a problem ;) I agree technically an attachment but its stored in the DB and presented differently. I'm leaning towards leaving where it is tho.

Reply:
> Well I could've left the partial named `bubble` then this wouldn't be a problem ;)

I know I know 😔 ! I actually thought of the "bubble" as the whole thing, not a subsection of the correspondence "box". Whatevs.

> I agree technically an attachment but its stored in the DB and presented differently. I'm leaning towards leaving where it is tho.

I don't think it matters at this point, so yeah let's just leave it.

I think it's just a bit semantically weird in `request/_incoming_correspondence.html.erb` to have:

```erb
    <div class="correspondence_text">
      <%= render partial: 'request/attachments',
                 locals: { incoming_message: incoming_message } %>
    </div>
```

Feels like it should be:

```erb
  <%= render partial: 'request/attachments', locals: { incoming_message: incoming_message } %>
  
  <div class="correspondence_text">
     <!-- reduced prominence conditionals omitted -->
      <%= incoming_message.get_body_for_html_display(@collapse_quotes) %>
  </div>
```

Reply:
We're currently rendering attachments within "correspondence_text" though, so yeah let's just leave it and unpick later.